hm1.py

```diff
@@ -19,4 +19,5 @@ def main():
                 print("-",key)
 
 if __name__ == '__main__':
-    main()
\ No newline at end of file
+    main()
+    
```

Comment:
Nothing much to tell you here. you wrote a good and simple algorithm that does the trick well. maybe I'd just put everything above lines 6-12 into a "buckets calculation" function and lines 3-4 into a read input function, just for good measure. Good Job.